tests/PCTCompile/test79/build.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<project name="PCTCompile-test79">
+  <taskdef resource="PCT.properties" />
+
+  <mkdir dir="build" />
+
+  <target name="test1">
+    <PCTCompile destDir="build" dlcHome="${DLC}" relativePaths="true" failOnError="false" outputType="json">
```

Comment:
failOnError is set to false in all cases, which is not the default. Why ?
Have at least one test case with failOnError="true", as your modification *may* have an impact on that.

Reply:
Correct me if I am wrong,
I think failOnError="true" will not impact my modification.
It will create the project-result.json file anyways. 
I think  handling failOnError="true" case would redundant

Reply:
I just want to make sure there won't be any regression in the future, so I want to verify that the file is present in case of success or failure, and for any value of failOnError.